test/config/naive_trial/naive_trial.py

```diff
@@ -0,0 +1,14 @@
+import time
```

Comment:
as you added naive_trial.py, what is the original trial.py used for?

Reply:
It is still being used by integration test.